Engine/ResourceManagement/Resources/Mesh.cpp

```diff
@@ -19,6 +19,11 @@ GLuint Mesh::GetVAO() const
 	return vao;
 }
 
+int Mesh::GetNumTriangles() const
+{
+	return vertices.size() / 3;
+}
+
```

Comment:
I am not 100% sure but, it shouldn't be indices.size() / 3? because maybe some vertices could be used more than once?

Reply:
oh fuck :D you are right!